### DISCUSSION TITLE
Export 3MF file including layer heights for 3d printers

### DISCUSSION THREAD

**Discussion by bramd** [skarkkai/touch-mapper#45] [Ideas]
3MF is an alternative file format to STL. The advantage of 3MF is that it can store more data. From what I gathered, it should also be able to store prefered layer heights for a model. It would be nice if TouchMapper can export this, so a blind person can just load it in a slicer and print, without modifying (inaccessible) settings.

**Comment by skarkkai** [comment]
I don't know if this is relevant anymore. Nonetheless, thanks for explaining the motivation. I agree that slicer UIs can be inaccessible, and “load and print without fiddling with settings” is a real problem for blind users.

That said, exporting slicer or printer settings (for example preferred layer heights) is out of scope for Touch Mapper. Those settings are highly slicer-, printer-, and material-specific, and I don’t have the expertise or capacity to maintain them. Touch Mapper deliberately focuses on generating tactile geometry; print configuration belongs on the slicer side.

What could make sense instead is a community-maintained approach: for example, a discussion thread where users share slicer profiles or workflows that work well for printing Touch Mapper output. I’m happy to provide a place for that kind of exchange, but I won’t be maintaining or endorsing printer/slicer profiles myself.

Would that kind of community discussion be useful for you?
